src/SourceCode.Clay.OpenApi/Builders/SchemaBuilder.cs

```diff
@@ -122,6 +125,13 @@ public class SchemaBuilder
         /// </summary>
         public SchemaBuilder()
         {
+            Enum = new List<ScalarValue>();
+            AllOf = new List<Referable<Schema>>();
+            OneOf = new List<Referable<Schema>>();
+            AnyOf = new List<Referable<Schema>>();
+            Not = new List<Referable<Schema>>();
+            Properties = new Dictionary<string, Referable<Schema>>();
```

Comment:
StringComparer.Ordinal

Reply:
I will have to write a comparer based on the spec. This will be updated as part of #80.